Documentation/ceph-csi-drivers.md

```diff
@@ -22,8 +22,9 @@ cluster.
    server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/)
 3. An up and running Rook instance (see [Rook - Ceph quickstart
    guide](https://github.com/rook/rook/blob/master/Documentation/ceph-quickstart.md))
+   (e.g. kubectl create -f common.yaml, operator.yaml, cluster.yaml)
 
-## CSI Drivers Enablement
+## CSI Plugin Deployment and Configuration 
```

Comment:
There is a link on line 12 that needs to be updated since this title changed

Reply:
Arg!  Nice catch!